filename_in: "/box_004010020/D_062110094/box_estimate_5"
dx_box: -0.977
dy_box: 0.0
dz_box: 0.0
gap_x: 0.0
gap_y: 0.01
gap_z: 0.0
gap_up: 0.2
hand_up: 0.3
duration: 3.0
duration_grasping: 3.0
string_tf: "/vito_anchor"